data/json/effect_on_condition.json

```diff
@@ -1298,5 +1298,24 @@
       { "u_add_trait": "CHITIN2" },
       { "u_add_trait": "CRUSTACEAN_CARAPACE" }
     ]
+  },
+  {
+    "type": "effect_on_condition",
+    "id": "dermatik_pregnancy_early",
+    "condition": { "u_has_effect": "dermatik" },
+    "effect": [
+      { "u_add_effect": "formication", "duration": "15 minutes" },
+      { "queue_eocs": "dermatik_pregnancy", "time_in_future": "1 days" }
+    ]
+  },
+  {
+    "type": "effect_on_condition",
+    "id": "dermatik_pregnancy",
+    "condition": { "u_has_effect": "dermatik" },
```

Comment:
Redundant. Change EOC type to ACTIVATION to prevent accidental double-ups and insure it only fires when told to